wiki/content/deploy/metrics.md

```diff
@@ -7,12 +7,13 @@ weight = 15
 +++
 
 
-Dgraph metrics follow the [metric and label conventions for
-Prometheus](https://prometheus.io/docs/practices/naming/).
+Dgraph database provides metrics on disk activity, memory usage, Dgraph instance activity
+and server node health - along with built-in metrics provided by Go. Dgraph metrics follow
```

Comment:
I would avoid using the "dash" inside a sentence, I suggest replacing it with a coma or semicolon.

`.. and server node health, along with built-in metrics...`